uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.component.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash'
+], function (_) {
+  return {
+    __name: 'calendarFeedsLinkModal',
+    bindings: {
+      dismiss: '<',
```

Comment:
Why do we need to pass `dismiss`? Wouldn't the behaviour of the X icon and Done button be always the same (just dismissing the modal and nothing else)?